.github/pull_request_template.md

```diff
@@ -1,13 +1,23 @@
-[ATTENTION. This is the *Pull Request Template*. Replace the two following lines with the indicated information and delete this one. Do not delete anything else]
-
 Closes #[issue number]
 [Description of the changes proposed in the pull request. Include steps to replicate the behavior and screenshots if UI is updated]
 
 # Review checklist
--   [ ] Terms and conditions reflect the current change
--   [ ] Contains enough appropriate tests
--   [ ] If aimed at production, writes a new summary in `whatsnew/whatsnew-pt-PT`
--   [ ] Properly adds an entry in `changelog.md` with the change
--   [ ] If PR includes UI updates/additions, its description has screenshots
--   [ ] Behavior is as expected
--   [ ] Clean, well-structured code
+
+## View Changes
+
+- [ ] Tested both in light and dark mode
+- [ ] Description has screenshots of the UI changes
+- [ ] Text is both in portuguese and english.
+
+## Documentation
+
+- [ ] Properly adds an entry in `changelog.md` with the change
```

Comment:
This is currently not used, I'm not sure we should keep it...

Reply:
Not used indeed. It could be automated by an action. That's a discussing

Reply:
Unless we force everyone to use [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) and therefore we can use _Conventional Changelogs_ there's no real way to automate this in a good enough way. 

I think that just appending the PR name/link to the `CHANGELOG` is not good enough.

Reply:
All the commits are too overwhelming

Reply:
That's the advantage of conventional commits: you can, for example, exclude `chore:` commits from the changelog or even exclude `fix:` commits, and it makes the changelog more readable. The bad thing about conventional commits is that you need to enforce it on every developer to work properly.

Reply:
An approach I saw when working with JavaScript monorepos is to use [changesets](https://github.com/changesets/changesets). It is a cli that allows each PR to create its own changeset, which contains a description of the user-facing changes made by the PR. When we release a new version, it can collect all changesets and create a changelog.

It also comes with a [Github Action](https://github.com/changesets/action) to ensure every PR has a changeset (there is a way to disable this for certain PRs, I think).

It can also handle semver, since each PR must specify if it's a major, minor or patch change.

Reply:
This is what a PR with changesets looks like: https://github.com/changesets/changesets/pull/1367/files#diff-4bc92bf808385e5c5386c204f0edb7e70e5c0c118677a53ea98cc6714461df89

Reply:
Is there a way to _force_ changesets before merging?

Reply:
changesets seems to be more annoying for the devs. The @LuisDuarte1 approach seems more automatic, but yes, requires forcing a rule

Reply:
And like @LuisDuarte1 said, Conventional Commits can be used but, in my experience with trying to get other people to use conventional commits, they are not trivial to start using...

Another aspect is that I think Conventional Commits are more tailored to technical projects with more technical changelogs, whereas changesets are more tailored for user-facing projects.

Reply:
Ultimately, in my opinion, this comes down to whether we want to make it easy for developers and generate mid-quality changelogs, possibly having to tweak them a little bit, or whether we want to make it slightly harder for developers and generate high-quality changelogs that don't require tweaking.

Reply:
I think we should remove this file and start creating github releases with the changelog